castai/resource_rebalancing_schedule.go

```diff
@@ -248,7 +255,10 @@ func stateToSchedule(d *schema.ResourceData) (*sdk.ScheduledrebalancingV1Rebalan
 			return nil, fmt.Errorf("parsing selector: %w", err)
 		}
 
-		keepDrainTimeoutNodes := readOptionalValue[bool](launchConfigurationData, "keep_drain_timeout_nodes")
+		evictGracefully := readOptionalValue[bool](launchConfigurationData, "evict_gracefully")
+		if evictGracefully == nil || *evictGracefully == false {
```

Comment:
this place is a hack as TF initializes both "evict_gracefully" and "keep_drain_timeout_nodes" as part of the nested struct.